lib/convection/model/template/resource_property/aws_cloudfront_viewercertificate.rb

```diff
@@ -9,6 +9,7 @@ class ResourceProperty
         class CloudFrontViewerCertificate < ResourceProperty
           property :use_default, 'CloudFrontDefaultCertificate'
           property :iam_certificate, 'IamCertificateId'
+          property :acm_certificate, 'AcmCertificateArn'
```

Comment:
According to `egrep -r 'property .+Arns?' lib`, the typical standard is that something like 'AcmCertificateArn' would become acm_certificate_arn.

Reply:
makes sense, will update.

Reply:
Done